tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -347,6 +357,16 @@ MakeResultPtr(const std::shared_ptr<T> &r, RLoopManager &lm, std::shared_ptr<RDF
 {
    return RResultPtr<T>(r, &lm, std::move(actionPtr));
 }
+
+template <typename T>
+std::unique_ptr<RMergeableValue<T>> GetMergeableValue(RResultPtr<T>& rptr){
```

Comment:
please add a docstring, and in particular specify that `GetMergeableValue` triggers the event loop that produces the RResultPtr's value if needed

Reply:
Absolutely! I will add a big commit with the docs in the end

Reply:
docs added